Tests/ClientRuntimeTests/NetworkingTests/Http/CRTClientEngineIntegrationTests.swift

```diff
@@ -32,7 +32,7 @@ class CRTClientEngineIntegrationTests: NetworkingTestUtils {
         var headers = Headers()
         headers.add(name: "Content-type", value: "application/json")
         headers.add(name: "Host", value: "httpbin.org")
-        let request = SdkHttpRequest(method: .get, endpoint: Endpoint(host: "httpbin.org", path: "/get", headers: headers))
+        let request = SdkHttpRequest(method: .get, endpoint: try Endpoint(host: "httpbin.org", path: "/get", headers: headers))
```

Comment:
Might be missing something, but I don't think this ctor throws

Reply:
There's a few places `try` is being used, by the way

Reply:
Removed all the `try`s